cmd/notation/cert/show.go

```diff
@@ -33,6 +33,14 @@ func certShowCommand(opts *certShowOpts) *cobra.Command {
 			opts.cert = args[0]
 			return nil
 		},
+		Long: `Show details of a certain certificate file
+
+Example - Show details of certificate "cert1.pem" with type "ca" from trust store "acme-rockets":
+  notation cert show --type ca --store "acme-rockets" cert1.pem
+
+Example - Show details of certificate "cert2.pem" with type "signingAuthority" from trust store "wabbit-networks":
```

Comment:
Same question. I cannot find type of "signingAuthority" in the spec https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md

Reply:
Yes we need to update that spec. Its mentioned in https://github.com/notaryproject/notaryproject/blob/main/specs/signing-scheme.md#trust-stores

Reply:
@priteshbandi Thanks. I will create an issue to track this spec update.